Source/WebCore/css/DeprecatedCSSOMValueList.h

```diff
@@ -32,26 +32,26 @@ namespace WebCore {
     
 class DeprecatedCSSOMValueList : public DeprecatedCSSOMValue {
 public:
-    static Ref<DeprecatedCSSOMValueList> create(const CSSValueList& value, CSSStyleDeclaration& owner)
+    static Ref<DeprecatedCSSOMValueList> create(const CSSValueList& values, CSSStyleDeclaration& owner)
     {
-        return adoptRef(*new DeprecatedCSSOMValueList(value, owner));
+        return adoptRef(*new DeprecatedCSSOMValueList(values, owner));
     }
-    
+
     String cssText() const;
-    
+
     size_t length() const { return m_values.size(); }
     DeprecatedCSSOMValue* item(size_t index) { return index < m_values.size() ? m_values[index].ptr() : nullptr; }
 
 private:
-    DeprecatedCSSOMValueList(const CSSValueList& value, CSSStyleDeclaration& owner)
-        : DeprecatedCSSOMValue(DeprecatedValueListClass, owner)
+    DeprecatedCSSOMValueList(const CSSValueList& values, CSSStyleDeclaration& owner)
+        : DeprecatedCSSOMValue(ClassType::List, owner)
     {
-        m_valueSeparator = value.separator();
-        m_values.reserveInitialCapacity(value.length());
-        for (unsigned i = 0, size = value.length(); i < size; ++i)
-            m_values.uncheckedAppend(value.itemWithoutBoundsCheck(i)->createDeprecatedCSSOMWrapper(owner));
+        m_valueSeparator = values.separator();
+        m_values.reserveInitialCapacity(values.length());
```

Comment:
```c++
m_values = WTF::map(values, [] (auto& value) { return value.createDeprecatedCSSOMWrapper(owner); });
```

(or something to that nature? might need to give CSSValueList a size() function if it doesn't have one).

Reply:
That’s a neat idea. Let me give it a try.